apps/data-types.md

```diff
@@ -358,7 +358,7 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | orderId | string | The unique identifier of the order. |
 | documentType | string | The type of the order document. Can be `Invoice` or `CreditNote`. |
 | documentNumber | string | The number of the order document. |
-| issueDate | string | The date/time specified on the document. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z` |
+| issueDate | string | The date/time specified on the document. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z` | Note: This date/time is in UTC and should be converted to the Timezone of the Issuer to match the date on the document |
```

Comment:
I understand what you're aiming at here, but the explanation is a bit confusing.
UTC is a time standard whereas GMT is a timezone. As far as I understand it, you cannot convert a time standard to a timezone. Let's talk about it.

Reply:
UTC is not a timezone because no country uses it.
It does represent a point in time that can be converted to a timezone.

https://www.timeanddate.com/time/gmt-utc-time.html
https://en.wikipedia.org/wiki/Coordinated_Universal_Time#Time_zones

Reply:
Yes, that's what I meant :D We'll talk later.